code/modules/client/preferences_gear.dm

```diff
@@ -233,6 +233,7 @@ var/global/list/gear_datums = list()
 	cost = 1
 	slot = slot_head
 
+/*
 /datum/gear/santahat
```

Comment:
This may cause issues with runtimes upon trying to spawn if people have the item saved.